server/apps/pa_img/pa_img_datalayer.py

```diff
@@ -76,13 +96,18 @@ def find(self, resource, req, lookup):
         :return:
         """
 
-        url = self._app.config['PAIMG_SEARCH_URL'] + '/search'
+        url = self._app.config['PAIMG_SEARCH_URL']
         fields = {}
         if 'query' in req['query']['filtered']:
-            query_keywords = req['query']['filtered']['query']['query_string']['query']
-            query_keywords = query_keywords.replace('slugline:', 'keywords:')
-            query_keywords = query_keywords.replace('description:', 'caption:')
-            fields = extract_params(query_keywords, ['headline', 'caption', 'keywords'])
+            query = req['query']['filtered']['query']['query_string']['query'] \
+                .replace('slugline:', 'keywords:') \
+                .replace('description:', 'caption:')
+            for key in ('headline', 'keywords', 'caption'):
```

Comment:
A query like this will not work, (all) will be ignored:
'(all) headline:(head one) caption:(capt) keywords:(key)'